packages/commonwealth/client/scripts/state.ts

```diff
@@ -238,6 +238,25 @@ const app: IApp = {
   cachedIdentityWidget: null,
 };
 
+function handleLoginSockets(data: any) {
```

Comment:
We should properly type data here.

Reply:
Not sure how we would do that? Should we move all the status.ts types into a common-common types file?

Reply:
i made the type more explicit like updateActiveUser, but this is also any

Reply:
by adding data.result.user